pkg/common/providers/ocmprovider/cluster.go

```diff
@@ -354,6 +354,114 @@ func (o *OCMProvider) ClusterKubeconfig(clusterID string) ([]byte, error) {
 	return []byte(resp.Body().Kubeconfig()), nil
 }
 
+// GetMetrics gathers metrics from OCM on a cluster
+func (o *OCMProvider) GetMetrics(clusterID string) (*v1.ClusterMetrics, error) {
+	clusterMetricsBuilder := v1.NewClusterMetrics()
+
+	clusterClient := o.conn.ClustersMgmt().V1().Clusters().Cluster(clusterID)
+
+	/*cluster, err := clusterClient.Get().Send()
+	if err != nil {
+		log.Printf("Error retrieving cluster: %s", err.Error())
+	}
+	*/
+
+	metricsClient := clusterClient.MetricQueries()
+	alertsMetricQuery, err := metricsClient.Alerts().Get().Send()
```

Comment:
We should wrap this in retryer logic to make sure this doesn't explode when we hit 500s.